docs/documentation/stories/disk-serve.md

```diff
@@ -0,0 +1,23 @@
+# Serving from Disk
+
+The CLI supports running a live reload experience to users by running `ng serve`. This will compile the application upon file saves and reload the browser with the newly compiled application. This is done by hosting the application in memory and serving it via [webpack-dev-server](https://webpack.js.org/guides/development/#webpack-dev-server).
+
+If you wish to get a similar experience with the application output to disk please use the steps below.
+
+## Environment Setup
+### Install a web server
+Because you will not be using webpack-dev-server you will need to install a web server for the browser to request the application. There are many to choose from but a good one to try is [live-server](https://www.npmjs.com/package/live-server) as it will auto-reload your browser when new files are outputed.
```

Comment:
"Because you will not be using webpack-dev-server you will need" --> "You will not be using webpack-dev-server , so you will need"

dont end or start sentence with preposition, when it can be avoided.